capsules/extra/src/button_keyboard.rs

```diff
@@ -127,12 +127,6 @@ impl kernel::hil::keyboard::KeyboardClient for ButtonKeyboard<'_> {
                 for (active_key_index, active_key) in self.key_codes.iter().enumerate() {
                     // If there is a match then we may want to handle this key.
                     if key == active_key {
```

Comment:
OK, just removing a debug that maybe shouldn't have made it into master to begin with?